VideoWeb/VideoWeb.UnitTests/Controllers/ConferenceManagement/DismissParticipantTests.cs

```diff
@@ -162,23 +190,25 @@ public async Task should_return_accepted_when_participant_is_a_room()
         {
             var judge = TestConference.GetJudge();
             var room = TestConference.CivilianRooms.First();
-            var witness = TestConference.Participants.First(x => x.IsWitness() && room.Participants.Contains(x.Id));
+            var participant = TestConference.Participants.First(x => x.IsWitness() && room.Participants.Contains(x.Id));
             var user = new ClaimsPrincipalBuilder()
                 .WithUsername(judge.Username)
                 .WithRole(AppRoles.JudgeRole).Build();
             Controller = SetupControllerWithClaims(user);
 
-            var result = await Controller.DismissParticipantAsync(TestConference.Id, witness.Id);
+            string expectedBody = $"{participant.HearingRole} dismissed by {judge.HearingRole}";
```

Comment:
Yes. Staff member could as well. Do you think that should be done in these tests?

Reply:
I looked at doing this. But the implementation for calling witnesses was not done so this would need to be done. I know the AC states dual host but is there other tickets for that functionality?

Reply:
I believe the story mention that in the subject. I don't have an intention to block this ticket with Dual Host/Staff member is not released with 1.28 as well. So we can create another 1 point Story to cover Staff Member scenario so that it is tracked and we push forward with this story to QA. Hope that's fine @simonwkelso

Reply:
As discussed this will be covered by VIH-8131